subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -94,6 +94,12 @@ static bool is_enabled(const struct bt_mesh_light_ctrl_srv *srv)
 	return srv->lightness->ctrl == srv;
 }
 
+static int32_t delay_remaining(struct bt_mesh_light_ctrl_srv *srv)
```

Comment:
Shouldn't it return `uint32_t` since `int32_t` causes implicit type conversion later in comparison and mathematic equations?